sumologic/sumologic_client.go

```diff
@@ -132,8 +134,10 @@ func (s *Client) GetWithCookies(urlPath string, cookies []*http.Cookie) ([]byte,
 func (s *Client) Post(urlPath string, payload interface{}) ([]byte, error) {
 	relativeURL, _ := url.Parse(urlPath)
 	sumoURL := s.BaseURL.ResolveReference(relativeURL)
-
+	log.Printf("[INFO] Post URL: %s", sumoURL)
```

Comment:
Do we need these log lines? If yes, maybe they should be at DEBUG level?

Reply:
Sorry my bad, I should have not commit that file; used those in my local testing, sorry
rollback that under https://github.com/SumoLogic/terraform-provider-sumologic/pull/285/commits/0d575a991ddbda1283d623aaa6b8975fac0067e7